composer.json

```
@@ -68,12 +73,32 @@
 	"autoload-dev": {
 		"psr-4": { "Imagify\\Tests\\": "Tests/"	}
 	},
+	"extra": {
+		"mozart": {
+			"dep_namespace": "ImagifyPlugin\\Dependencies\\",
+			"dep_directory": "/src/Dependencies/",
+			"classmap_directory": "/classes/Dependencies/",
```

```suggestion
			"classmap_directory": "/inc/classes/Dependencies/",
```
Not-namespaced classses are in `inc/classes`, we can add the not-namespaced dependencies there too